docs/learning/tutorials/develop-desktop-viewer.md

```diff
@@ -1,11 +1,13 @@
 # Developing a Desktop Viewer
 
+The application is built using [Vite](https://github.com/vitejs/vite), and scaffolded using [degit](https://github.com/Rich-Harris/degit).
+
 ## Setup
 
 - [Install necessary prerequisites](./development-prerequisites.md).
-- From a terminal, `npx create-react-app your-app-name --template @itwin/desktop-viewer --scripts-version @bentley/react-scripts`
+- From a terminal, `npx degit iTwin/viewer/packages/templates/desktop#master your-app-name`
   - This will generate a new application based on the iTwin Viewer React component in the `your-app-name` directory.
-- Open the `your-app-name` directory in VS Code
+- Open the `your-app-name` directory in VS Code and run `npm install`. This will install all the dependencies listed in your package.json.
```

Comment:
```suggestion
- Open the `your-app-name` directory in VS Code and run `npm install` (or `pnpm install`). This will install all the dependencies listed in your package.json.
```